### ISSUE TITLE
virtualbox-ose-guest-dkms-5.2.18_1 fails to build kernel modules.

### ISSUE THREAD

**Issue by crocket** [void-linux/void-packages#2726]
### System

* xuname: `Void 4.18.7_2 x86_64 AuthenticAMD/KVM notuptodate rFF`
* package: `virtualbox-ose-guest-dkms-5.2.18_1`

### Expected behavior

`xbps-reconfigure -f virtualbox-ose-guest-dkms` should compile kernel modules.

### Actual behavior

```
$ sudo xbps-reconfigure -f virtualbox-ose-guest-dkms
virtualbox-ose-guest-dkms: configuring ...
DKMS module 'virtualbox-ose-guest-5.2.18' already added, skipping.
Building DKMS module 'virtualbox-ose-guest-5.2.18' for kernel-4.18.7_2... FAILED!
DKMS module 'virtualbox-ose-guest-5.2.18' failed to build, please check /var/lib/dkms
for errors in the log file.
virtualbox-ose-guest-dkms: configured successfully.
```

/var/lib/dkms/virtualbox-ose-guest/5.2.18/build/[make.log](https://github.com/void-linux/void-packages/files/2381406/make.log)

### Steps to reproduce the behavior

```
xbps-reconfigure -f virtualbox-ose-guest-dkms
```

### Related Issues

* https://www.virtualbox.org/ticket/17981

**Comment by crocket** [comment]
https://www.virtualbox.org/ticket/17981

**Comment by abenson** [comment]
Just to verify, do you have linux4.18-headers installed?

**Comment by Hoshpak** [comment]
fixed in 5.2.20.
